src/constants/ballEvolutions.ts

```diff
@@ -64,6 +65,39 @@ export const basicEvolutions: [StarterBalls, StarterBalls, Balls][] = [
   ['poison', 'wind', 'noxious'],
 ];
 
+export function getEvolutionRecipes(ball: Balls): string | null {
+  let matches = basicEvolutions
```

Comment:
`matches` is never reassigned, so `const` should be used
```suggestion
  const matches = basicEvolutions
```